tests/tests/cpld_control_gfsv17

```diff
@@ -78,19 +79,56 @@ export RESTART_N=3
 export RESTART_INTERVAL="${RESTART_N} -1"
 export OUTPUT_FH='0 21 24'
 
+export INPES=$INPES_cpl_unstr
```

Comment:
Is there a specific reason to use INPES_cpl_unstr instead of the default? I think only WAV_tasks needs to be cpld_unstr. Sorry I was working on testing the HR2 hanging issue on wcoss2 and did not have time to review the PR. Anyway, this may need a future PR to fix.

Reply:
In the other pdlib tests, this was done so I followed that template. @DeniseWorthen might be able to provide additional insights.  I'm sure there are ways to streamline and improve these tests and look forward to a future PR that does this.   I know we're all busy.  I appreciate you taking the time now and for your valuable feedback.

Reply:
@junwang-noaa I don't have any specific memory of why all the tasking values are using cpl_unstr. I was also probably mimicking what had been done previously.